test/commit-and-push.ts

```diff
@@ -330,55 +332,16 @@ describe('Commit and push function', async () => {
     });
   });
   it('Forwards GitHub error if getCommit fails', async () => {
-    // setup
-    const commitErrorMsg = 'Error committing';
-    sandbox.stub(octokit.git, 'getCommit').rejects(Error(commitErrorMsg));
-    try {
-      // tests
-      await handler.createTree(octokit, origin, '', []);
-    } catch (err) {
-      expect(err.message).to.equal(commitErrorMsg);
-    }
+    const error = new Error('Error committing');
+    sandbox.stub(octokit.git, 'getCommit').rejects(error);
+    await assert.rejects(handler.createTree(octokit, origin, '', []), error);
   });
   it('Forwards GitHub error if createTree fails', async () => {
     // setup
-    const createTreeErrorMsg = 'Error committing';
```

Comment:
Ok this requires an explanation :) With try/catch test pattern here, the intent was to ensure the code throws an error, and then check the error message.  That only works if you put an `assert.fail` after the code that you expect to throw.  Turns out ... these tests never worked as intended.  The `handleTree` method here and below never actually threw, so when I converted them to `assert.rejects`, the tests started failing.  Since they never worked as intended, I just went ahead and removed them.  We can come back around to improve coverage.